### PR TITLE
Fix typo on readme

### PR DESCRIPTION
**Pull request by asizon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Then you can make a build for a specific target configuration e.g.
 
 The `TIMER_MAP` define sets up the timers default settings, as adjusted with the `timer` and `dma` commands in CLI. The problem is there are some magic numbers (index entries), that need to be known.
  
-You can activate the `timer map` CLI command (by compiling with `USE_TIMER_MAP_PRINT` defined i.e. `"EXTRA_FLAGS=-DUSE_TIMER_MAP_PRINT"`).
+You can activate the `timer map` CLI command (by compiling with `USE_TIMER_MAP_PRINT` defined i.e. `"EXTRA_FLAGS=-USE_TIMER_MAP_PRINT"`).
```
Comment (haslinghuis) [inline README.md:30]: This is intentional:

`-D[FLAG][=VALUE]` is a preprocessor flag.

**Review by nerdCopter — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (asizon) [reply README.md:30]: Sorry muy bad

**Comment by asizon** [comment]
@nerdCopter aproving It now im confused

File: README.md (hunk 1 above)
Comment (haslinghuis) [reply README.md:30]: No worries - it's part of the make command.

**Comment by nerdCopter** [comment]
sorry, thought that haslinghuis was the PR creator.
`-D` is used for non-cloud-build, i misunderstood.

**Pull request closed without merging**